build.gradle

```diff
@@ -1,6 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.kotlin_version = '1.1.51'
```

Comment:
Version [1.1.60](https://blog.jetbrains.com/kotlin/2017/11/kotlin-1-1-60-is-out/) is already available